design/sketch.webidl

```diff
@@ -169,12 +169,12 @@ interface GPUTextureUsage {
 
 dictionary GPUTextureDescriptor {
     GPUExtent3D size;
-    u32 arrayLayerCount;
-    u32 mipLevelCount;
-    u32 sampleCount;
-    GPUTextureDimension dimension;
-    GPUTextureFormat format;
-    GPUTextureUsageFlags usage;
+    u32 arrayLayerCount = 1;
+    u32 mipLevelCount = 1;
+    u32 sampleCount = 1;
+    GPUTextureDimension dimension = "2d";
+    GPUTextureFormat format = "rgba8unorm";
```

Comment:
I don't think this is an obvious default even though it's a little more common than the other options. Probably most calls will set it anyway.

Reply:
Looks like [D3D12 doesn't](https://docs.microsoft.com/en-us/windows/desktop/direct3d12/cd3dx12-resource-desc) have a default, whereas [Metal does](https://developer.apple.com/documentation/metal/mtltexturedescriptor/1515450-pixelformat).